thorbanks/utils.py

```diff
@@ -1,41 +1,64 @@
-from __future__ import unicode_literals
-
+from base64 import b64decode, b64encode
 from functools import reduce
-from base64 import b64encode, b64decode
-import hashlib
 
 from django.urls import reverse
 from django.utils.encoding import force_str
 
-from Crypto.Hash import SHA
-from Crypto.Signature import PKCS1_v1_5
-from Crypto.PublicKey import RSA
+from cryptography.exceptions import InvalidSignature
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import hashes, serialization
+from cryptography.hazmat.primitives.asymmetric import padding
 
 from thorbanks import settings
 
 
 IPIZZA_REQUEST_ORDER = {
-    '3012': (
-        'VK_SERVICE', 'VK_VERSION', 'VK_USER',
-        'VK_DATETIME', 'VK_SND_ID', 'VK_REC_ID',
-        'VK_USER_NAME', 'VK_USER_ID', 'VK_COUNTRY',
-        'VK_OTHER', 'VK_TOKEN', 'VK_RID',
+    "3012": (
```

Comment:
Fishy magic constants ...

Reply:
All of this is explained in the iPizza protocol. Afaik there is no documentation for it in English. 

More info here - https://banks.maximum.thorgate.eu/et/info (see links to specific bank documentations)

Reply:
@Jyrno42 Well regardless of any external documentation available, these do look like magic constants in this library :) I guess they still got some meaningful names that could be used as constant names -- especially when there is some comparison happening below.

This is not a nitpick on this PR, but a general point which could be improved later.

Reply:
In my opinion adding our own abstraction to these domain specific values makes little sense. I would even argue that it would cause extra confusion since currently one can take a look at the protocol documentation and instantly figure out what these mean. But if we make up our own names for these we will have to:

- document them
- have some translation code which can give us a "service code" from the "pretty name"

Another note:

These values are not really exposed to the users of this library (unless they are already familiar with the banklink protocol, e.g. they already know of a specific `VK_SERVICE` code they need to use).